site/src/examples/progress/LinearIndeterminate.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactElement, useEffect, useState } from "react";
+import {
+  Button,
+  LinearProgress,
+  LinearProgressProps,
+  Toast,
+  ToastContent,
+  ValidationStatus,
+  Text,
+} from "@salt-ds/core";
+import { CloseIcon } from "@salt-ds/icons";
+
+export const LinearIndeterminate = (): ReactElement => {
+  const [variant, setVariant] =
+    useState<LinearProgressProps["variant"]>("indeterminate");
+
+  const [value, setValue] = useState(0);
+
+  const [toastStatus, setToastStatus] = useState<{
+    header: string;
+    status: ValidationStatus;
+    message: string;
+  }>({
+    header: "File uploading",
+    status: "info",
+    message: "File upload to shared drive in progress.",
+  });
+
+  useEffect(() => {
+    const timer = setTimeout(() => setVariant("determinate"), 4000);
+    return () => clearTimeout(timer);
+  }, []);
+
+  useEffect(() => {
+    if (variant === "determinate" && value < 100) {
+      const interval = setInterval(
+        () => setValue((currentValue) => currentValue + 1),
+        20
+      );
+      return () => clearInterval(interval);
+    }
+  }, [value, variant]);
+
+  useEffect(() => {
+    if (value === 100) {
+      setToastStatus({
+        header: "Upload complete",
+        status: "success",
+        message: "File has successfully been uploaded to shared drive.",
+      });
+    }
+  }, [value]);
+
+  return (
+    <Toast status={toastStatus.status}>
```

Comment:
@joshwooding I've added the Toast example to the site, did you also want to change it?

Reply:
Is this a new pattern we are trying to create ?

When I look at this I think the Toast is part of the Progress Bar, where did my Progress bar go ?
Is this confusing to the product or design folks, who do not look at the code, they will say, that the final state of the indeterminate Progress bar replaces itself with something that looks like Toast.

I don't actually see the point of this example, you want to show it going from indeterminate to determinate but after that it's just the same as any other Progress bar example ?

Reply:
I think we will need to discuss with design cc @navkaur76 @bhoppers2008